crates/robonode-client/src/authenticate.rs

```diff
@@ -54,16 +57,29 @@ pub enum AuthenticateError {
     /// The person was not found, it is likely because they haven't enrolled first.
     #[error("person not found")]
     PersonNotFound,
+    /// The person was not found, it is likely because they haven't enrolled first, returned blob.
+    #[error("person not found, returned blob")]
+    PersonNotFoundReturnedBlob(ScanResultBlob),
```

Comment:
I suggest we call this one `PersonNotFound(ScanResultBlob),`, and rename the old one to `PersonNotFoundNoBlob`

Reply:
Do we still plan to support the old robonodes?

Reply:
Right, forward compatibility on `robonode-server` side, but back-forward compatibility on `robonode-client` side.

Reply:
https://github.com/humanode-network/humanode/pull/1026/commits/d9e6b0f831ac9ca3752f4e8716364847fba33cad